ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ChainDB/Model/Test.hs

```diff
@@ -93,3 +94,10 @@ prop_between_currentChain bt =
     from     = StreamFromExclusive genesisPoint
     to       = StreamToInclusive $ M.tipPoint model
     secParam = protocolSecurityParam singleNodeTestConfig
+
+{-------------------------------------------------------------------------------
+  Orphan instances
+-------------------------------------------------------------------------------}
+
+instance ModelSupportsBlock TestBlock where
+  isEBB = const IsNotEBB
```

Comment:
A bit annoying: this TestBlock lives in `test-util`, so it can implements `ModelSupportsBlock` (`test-consensus`) itself, hence the orphans :disappointed:.